_posts/2018-12-14-rediscovering-hosts-via-grub2.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: 
+date: 2018-12-14 12:25:27
+author: Foreman
+tags:
+- foreman
+- discovery
+---
+
+When a host is managed via Foreman, it's possible to download discovery image
+to it and create a Grub2 boot entry to quickly enter discovery mode again. This
+ad-hoc workflow requires the old managed host entry to be deleted prior
```

Comment:
maybe we can add a hammer host delete command to the script?

Reply:
It's actually not mandatory, only for older versions of Foreman. However it's recommended otherwise you will end up with old records in the inventory for no reason. I've added a note.